psydac/api/postprocessing.py

```diff
@@ -779,17 +913,30 @@ def load_snapshot(self, n, *fields):
                         index = tuple(slice(s, e + 1) for s, e in zip(Vi.starts, Vi.ends))
 
                         new_field.coeffs[i][index] = coeff[index]
+                    new_field.coeffs.update_ghost_regions()
                     self._snapshot_fields[field_name] = new_field
 
         self._loaded_t = snapshot_group.attrs['t']
         self._loaded_ts = snapshot_group.attrs['ts']
-        fh5.close()
 
         for key in list(self._snapshot_fields.keys()):
             if key not in fields:
                 del self._snapshot_fields[key]
 
-    def export_to_vtk(self, filename_pattern, grid, npts_per_cell=None, snapshots='none', lz=4, fields=None, debug=False):
+        self._last_loaded_fields = self._snapshot_fields
+
+    def export_to_vtk(self, 
+                      filename_pattern, 
+                      grid, 
+                      npts_per_cell=None, 
+                      snapshots='none', 
+                      lz=4, 
+                      logical_grid=False, 
+                      fields=None, 
+                      additional_physical_functions=None,
+                      additional_logical_functions=None,
+                      number_by_rank=True,
+                      debug=False):
```

Comment:
This function takes a great number of parameters, hence I recommend that you make all the ones that have a default value into keyword-only arguments, by adding a `*` after `grid`.

Reply:
Added a star in [b8b1cea](https://github.com/pyccel/psydac/pull/214/commits/b8b1cea2241f1cb78baccab3fd96192b277d57f5)